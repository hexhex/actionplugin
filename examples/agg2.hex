% -*- prolog -*-

% testA returns "foo" if the extension of its input predicate is empty
% testA returns "bar" if the extension of its input predicate has one member

c(1).
pred(c).
pred(d).

in(X) :- &testA[P](X), pred(P).

num_in(N) :- #count{ X : in(X) } = N.