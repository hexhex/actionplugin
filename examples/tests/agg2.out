{num_in(2), in(bar), in(foo), pred(d), pred(c), c(1)}
